tests/__init.test.js

```diff
@@ -28,7 +28,7 @@ setup(function () {
 
 teardown(function () {
   // Clean up any attached elements.
-  ['canvas', 'a-assets', 'a-scene'].forEach(function (tagName) {
+  ['canvas', 'a-assets', 'a-scene', '#aframe-inspector'].forEach(function (tagName) {
```

Comment:
OK, I would just set a `url` property on the inspector component in the test to a dummy script URL, rather than loading from the real URL.